### PR TITLE
Enhanced App UI Sizes Organization

### PR DESCRIPTION
**Pull request by Aarush-Acharya**
To be done

- [x] Add additional planned changes

To be cleaned 

 
- [x] room_screen.dart
- [x] main.dart
- [x] pair_chat_dialog.dart
- [x] create_room_screen.dart
- [x] discussions_screen.dart
- [x] email_verification_screen.dart
- [X] home_screen.dart
- [X] connect to the backend and testing
- [X] login_screen.dart
- [x] onboarding_screen.dart
- [X] profile_screen.dart
- [x] signup_screen.dart
- [x] splash_screen.dart
- [x] tabview_screen.dart

### CODE REVIEW

**Comment by chandansgowda** [comment]
@Aarush-Acharya Let this PR be under draft until I get the pair chat feature ready. It will take two to three days max. Then please clean that also according to what we have done previously and then mark the PR for review.

**Comment by Aarush-Acharya** [comment]
Alright understood thanks

**Comment by Aarush-Acharya** [comment]
Hey @chandansgowda  I have cross-checked all the screens and everything works smoothly from my end this PR is ready for review please do look into it

**Comment by chandansgowda** [comment]
@Aarush-Acharya Please give a more descriptive name for the PR

File: assets/images/timmer.svg
Comment (chandansgowda) [inline assets/images/timmer.svg:1]: Correct typo in file name

**Review by chandansgowda — CHANGES_REQUESTED** [review]
1. The register now button should be moved to the bottom in login screen and login button should be in the bottom in registration screen.
Screenshot 2023-08-26 at 6 09 42 PM

2. Verify Email button gives this snackbar with conflicting title and response.
Screenshot 2023-08-26 at 6 10 55 PM

File: assets/images/timmer.svg
Comment (Aarush-Acharya) [reply assets/images/timmer.svg:1]: Hey @chandansgowda implemented  the requested changes

**Comment by Aarush-Acharya** [comment]
New Login Screen

Screenshot 2023-08-26 at 6 58 15 PM

New SignUp Screen

Screenshot 2023-08-26 at 6 59 37 PM


New Verify Email SnackBar

image

**Review by chandansgowda — APPROVED** [review]
Great job @Aarush-Acharya

**Comment by Aarush-Acharya** [comment]
thanks a lot @chandansgowda ]

**Comment by Aarush-Acharya** [comment]
Hey @chandansgowda please review this

**Review by chandansgowda — APPROVED** [review]
LGTM! 
@jddeep please review and merge.

**Pull request merged**